user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -370,12 +369,6 @@ def check_discount_percentage(discount_type: str, discount_value: int):
             self.ctxt.set_ctxt('cpu_discount', cpu_discount)
             self.ctxt.set_ctxt('gpu_discount', gpu_discount)
 
-    def _create_cluster_report_args(self) -> list:
```

Comment:
This is not required any more since we want to generate cluster information in every case.